keyboards/dotmod/dymium65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,93 @@
+    [_LN] = LAYOUT_split_space(
+    KC_ESC,                            
+                   KC_1,        KC_2,        KC_3,  KC_4,  KC_5,  KC_6,      KC_7,  KC_8,  KC_9,         KC_0,         KC_MINUS,        KC_EQUAL,         KC_BACKSPACE,   KC_AUDIO_VOL_UP,KC_AUDIO_VOL_DOWN,
+    KC_TAB,        KC_Q,        KC_W,        KC_E,  KC_R,  KC_T,  KC_Y,      KC_U,  KC_I,  KC_O,         KC_P,         KC_LEFT_BRACKET, KC_RIGHT_BRACKET, KC_BACKSLASH,            
+    KC_CAPS_LOCK,  KC_A,        KC_S,        KC_D,  KC_F,  KC_G,  KC_H,      KC_J,  KC_K,  KC_L,         KC_SEMICOLON, KC_QUOTE,                          KC_ENTER,       KC_PAGE_UP,     KC_PAGE_DOWN,
+    KC_LEFT_SHIFT,              KC_Z,        KC_X,  KC_C,  KC_V,  KC_B,      KC_N,  KC_M,  MO(_MT),      _______,      KC_SLASH,                          KC_RIGHT_SHIFT, KC_UP,          KC_AUDIO_MUTE,
+    KC_LEFT_CTRL,  KC_LEFT_GUI, KC_LEFT_ALT,                      KC_SPACE,                KC_RIGHT_ALT, _______,                       KC_RIGHT_CTRL,    KC_LEFT,        KC_DOWN,        KC_RIGHT
+    ),
+    [_MT] = LAYOUT_split_space(
+    KC_ESC,                            
+                   KC_1,        KC_2,        KC_3,  KC_4,  KC_5,  KC_6,      KC_7,  KC_8,  KC_9,         KC_0,         KC_MINUS,        KC_EQUAL,         KC_BACKSPACE,   KC_AUDIO_VOL_UP,KC_AUDIO_VOL_DOWN,
+    KC_TAB,        KC_Q,        KC_W,        KC_E,  KC_R,  KC_T,  KC_Y,      KC_U,  KC_I,  KC_O,         KC_P,         KC_LEFT_BRACKET, KC_RIGHT_BRACKET, KC_BACKSLASH,            
+    KC_CAPS_LOCK,  KC_A,        KC_S,        KC_D,  KC_F,  KC_G,  KC_H,      KC_J,  KC_K,  KC_L,         KC_SEMICOLON, KC_QUOTE,                          KC_ENTER,       KC_PAGE_UP,     KC_PAGE_DOWN,
+    KC_LEFT_SHIFT,              KC_Z,        KC_X,  KC_C,  KC_V,  KC_B,      KC_N,  KC_M,  _______,      _______,      KC_SLASH,                          KC_RIGHT_SHIFT, KC_UP,          KC_AUDIO_MUTE,
+    KC_LEFT_CTRL,  KC_LEFT_GUI, KC_LEFT_ALT,                      KC_SPACE,                KC_RIGHT_ALT, _______,                       KC_RIGHT_CTRL,    KC_LEFT,        KC_DOWN,        KC_RIGHT
+    )
+};
+
+ bool encoder_update_user(uint8_t index, bool clockwise) {
+    uint8_t  layer_new      = biton32(layer_state);
+    uint16_t ringt_key_code = keymap_key_to_keycode(layer_new,(keypos_t){.row = 1,.col = 14});
+    uint16_t left_key_code  = keymap_key_to_keycode(layer_new,(keypos_t){.row = 1,.col = 15});
+
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(ringt_key_code);
+        } else {
+            tap_code(left_key_code);
+        }
+    }
+    return true;
+}
+
+/* If you want to set custom indicators,
+   such as an LED for Caps Lock, or layer indication,
+   you can use the or function for that  */
+ void rgb_matrix_indicators_kb(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        rgb_matrix_set_color(LED_CAP_LOCK_INDEX, RGB_WHITE);
+    }
```

Comment:
Please be testing your changes... this will have caused a compilation error.
```suggestion
 bool rgb_matrix_indicators_kb(void) {
    if (host_keyboard_led_state().caps_lock) {
        rgb_matrix_set_color(LED_CAP_LOCK_INDEX, RGB_WHITE);
    }
    return false;
```